validator/remote/src/main/java/tech/pegasys/teku/validator/remote/apiclient/ValidatorApiMethod.java

```diff
@@ -23,7 +23,9 @@
   GET_VALIDATORS("eth/v1/beacon/states/head/validators"),
   GET_DUTIES("validator/duties"),
   GET_UNSIGNED_BLOCK_V2("eth/v2/validator/blocks/:slot"),
+  GET_UNSIGNED_BLINDED_BLOCK("eth/v1/validator/blinded_blocks/:slot"),
   SEND_SIGNED_BLOCK("eth/v1/beacon/blocks"),
+  SEND_SIGNED_BLINDED_BLOCK("eth/v1/beacon/blinded_blocks"),
```

Comment:
Did we actually use this new API anywhere?

Reply:
um.... no.... doing now. :/